lightstep/resource_metric_condition.go

```diff
@@ -194,7 +194,7 @@ func getMetricQuerySchema() map[string]*schema.Schema {
 		"display": {
 			Type:         schema.TypeString,
 			Optional:     true,
-			ValidateFunc: validation.StringInSlice([]string{"line", "area", "bar", "big_number", "heatmap"}, false),
+			ValidateFunc: validation.StringInSlice([]string{"line", "area", "bar", "big_number", "heatmap", "dependency_map"}, false),
```

Comment:
so there's a very unsatisfying answer to your question: NO `dependency_map` is not a valid display type for alert queries but YES it needs to be added here -- im pretty sure some legacy dashboards TF validation code calls through here

i have no idea what would happen if someone tried to make an alert with display type `dependency_map`... in the UI we only allow `line`. maybe there's some other validation code in here somewhere that does that validation for us? but regardless this is not a new problem (for instance heatmap makes just as little sense in an alert)

Reply:
I also wonder what happens with `big_number` given we deprecated that and it's `big_number_v2` now...